charts/osm/templates/osm-webhook-deployment.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: apps/v1
```

Comment:
Please rename this to `osm-validator` or something since `osm-webhook` does not imply what this webhook does.

Reply:
+1 I think `osm-webhook` needs to be renamed to something more meaningful